pkg/apis/monitoring/v1alpha1/scrapeconfig_types.go

```diff
@@ -22,9 +22,15 @@ import (
 )
 
 const (
-	ScrapeConfigsKind   = "ScrapeConfig"
-	ScrapeConfigName    = "scrapeconfigs"
-	ScrapeConfigKindKey = "scrapeconfig"
+	ScrapeConfigsKind           = "ScrapeConfig"
+	ScrapeConfigName            = "scrapeconfigs"
+	ScrapeConfigKindKey         = "scrapeconfig"
+	KubernetesRolePod           = "Pod"
+	KubernetesRoleEndpoint      = "Endpoint"
+	KubernetesRoleIngress       = "Ingress"
+	KubernetesRoleService       = "Service"
+	KubernetesRoleNode          = "Node"
+	KubernetesRoleEndpointSlice = "EndpointSlice"
```

Comment:
For enums, I feel like it's better to declare them together if their type, that way we can easily find the possible values when we randomly find the type through the code.

Could you move this down? Right below the `type KubernetesRole string`?